plugin/documents.py

```diff
@@ -507,6 +512,21 @@ def on_query_completions(self, prefix: str, locations: List[int]) -> Optional[su
             lambda: self._on_query_completions_async(completion_list, locations[0], triggered_manually))
         return completion_list
 
+    def _format_on_paste(self) -> None:
+        self.purge_changes_async()
+        clipboard_text = sublime.get_clipboard()
+        sel = self.view.sel()
+        number_of_cursors = len(sel)
+        split_clipboard_text = clipboard_text.split('\n')
+        for index, region in enumerate(sel):
+            look_text = clipboard_text
+            if len(split_clipboard_text) == number_of_cursors:
+                look_text = split_clipboard_text[index]
+            found_region = self.view.find(look_text, region.end(), sublime.REVERSE)
```

Comment:
currently LSP supports ST 4070
sublime.REVERSE was introduced in build 4149
if this PR gets merged the last supported version needs to be 4149.